src/Spout/Reader/XLSX/Helper/SharedStringsHelper.php

```diff
@@ -60,15 +60,28 @@ public function __construct($filePath, $tempFolder = null)
      */
     public function hasSharedStrings()
     {
-        $hasSharedStrings = false;
+        return $this->getSharedStringsPath() !== null;
+    }
+
+    /**
+     * Returns shared strings XML file path
+     *
+     * @return string|null
+     */
+    public function getSharedStringsPath()
```

Comment:
This function's name is confusing, as there already is a `getSharedStringsFilePath()` function in this file